packages/mds-provider/api.ts

```diff
@@ -199,6 +199,25 @@ function api(app: express.Express): express.Express {
 
   // / /////////////////////// trips /////////////////////////////////
 
+  const getStage0Properties = (items: { recorded: Timestamp; sequence?: number | null }[]) => {
```

Comment:
I'm not a huge fan of numbers in function names, but considering we don't have very strictly defined style rules yet I guess it's okay!

Reply:
These are properties that are not in the MDS spec. Adopted "Stage 0" for proposals which is similar to what TC39 does for ECMAScript.